clio-client/src/main/scala/org/broadinstitute/clio/client/metadata/ArrayDeliverer.scala

```diff
@@ -8,11 +8,16 @@ class ArrayDeliverer extends MetadataMover[ArraysMetadata] {
   override protected def moveMetadata(
     src: ArraysMetadata,
     destination: URI,
-    newBasename: Option[String]
+    newBasename: Option[String],
+    undeliver: Boolean
   ): (ArraysMetadata, Iterable[IoOp]) = {
     import MetadataMover.buildFilePath
 
-    val idatDestination = destination.resolve(ArrayDeliverer.IdatsDir)
+    val idatDestination = if (undeliver) {
+      destination.resolve("../").resolve(ArrayDeliverer.IdatsDir)
```

Comment:
Instead of doing it this way, how about making `IdatsDir` overridable and creating `ArrayUndeliverer` which subclasses `ArrayDeliverer` and overrides this field? That would be a smaller change and limit the scope to `UndeliverArrays`.

https://github.com/broadinstitute/clio/blob/develop/clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala#L390